### ISSUE TITLE
[LG] Multi Language supports loading lg file dynamically

### ISSUE THREAD

**Issue by ssss141414** [microsoft/botbuilder-dotnet#2736]
**Is your feature request related to a problem? Please describe.**
I split logic and text in two lg files like this:
```
Activity.lg

# Test
[Activity
    Text = {Test.Text()}
    InputHint = expectingInput
]
```
```
Text.lg

# Test.Text
- balabala
```
And a top-level file import all the lg files:
```
All.lg

[import](Activity.lg)
[import](Text.lg)
```
I use this top-level file as an 'entry':
```
this.UseResourceExplorer(resourceExplorer);
this.UseLanguageGeneration(resourceExplorer, "All.lg");
```

Now I hope Text.lg can support multi language, for example Text.zh-cn.lg. So that when I talk to my bot in Chinese, the bot will reply to me in Chinese. It can also save the time needed to translate lg files into different language(We only need to translate Text lg file).
```
Text.zh-cn.lg

# Test.Text
- 你好
```
But this doesn't work.

**Describe alternatives you've considered**
This will work but not very convenient: Add multi-lang All.lg file
```
All.zh-cn.lg

[import](Activity.lg)
[import](Text.zh-cn.lg)
```
This just like assembling a project. I should write this 'config file' to tell LG what files I used,  for all languages I support.
It seems that multi-lang in LG can load 'top-level' file dynamically(In this example it's All.lg). But I hope that I don't need to do this. And I hope that when I talk in Chinese, it can find Text.lg file in Chinese dynamically.

## [Update 10/22/2019] 
This workaround will not work in coming SDK. Because structured lg will also need [import]. This will cause a duplicate import error.

**Comment by vishwacsena** [comment]
We need to clean up and document multi-lingual support outside of declarative. 

@boydc2014  - let's discuss this in our next sync.

**Comment by boydc2014** [comment]
> We need to clean up and document multi-lingual support outside of declarative.
> 
> @boydc2014 - let's discuss this in our next sync.

Yes, definitely doc and samples should be updated. But this user is actually reporting one issue, that our "Multi-languageGenerator" only support dynamically choose lg file at top level. Which mean it can choose from "all.en_us.lg" and "all.zh_cn.lg", but if "all.en_us.lg" import "text.lg", it will not resolve to "text.en_us.lg".

**Comment by vishwacsena** [comment]
Presumed fixed via #2800
